From 9965ec1b9e6f6db192a2355e4e3eb80bc27fc092 Mon Sep 17 00:00:00 2001
From: Ahmad Samir <a.samirh78@gmail.com>
Date: Sun, 24 Jan 2021 11:10:27 +0200
Subject: [PATCH] Fix build with gcc11

Apparently now we need to #include <limit>.

Patch by Geoff Leach:
https://bugs.gentoo.org/766480
https://invent.kde.org/frameworks/kio/-/issues/3

NO_CHANGELOG
---
 src/kntlm/des.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/kntlm/des.cpp b/src/kntlm/des.cpp
index 18d85bfd..65c86f06 100644
--- a/src/kntlm/des.cpp
+++ b/src/kntlm/des.cpp
@@ -29,6 +29,8 @@
 #include <string.h>
 #include <qendian.h>
 
+#include <limits>
+
 static void permute_ip(unsigned char *inblock, DES_KEY *key, unsigned char *outblock);
 static void permute_fp(unsigned char *inblock, DES_KEY *key, unsigned char *outblock);
 static void perminit_ip(DES_KEY *key);
-- 
GitLab

